css/main.css

```diff
@@ -68,22 +68,26 @@ h1 {
   font-weight: bold;
   letter-spacing: -0.3px;
   color: rgb(255, 255, 255);
-  padding: 20px 51px 19px 57px;
+  padding: 20px 110px 19px 110px;
   background: #58c4c4;
   border: 0;
   border-radius: 3px;
   cursor: pointer;
 }
 
-.main_button:hover {
+.main-button_get-started {

```

Comment:
`.main-button__get-started` (extra dash `_` is needed)

Reply:
@probil, extra dash is not needed, because `get-started` is a modifier for button. (Button may have another text and other padding)

Reply:
[According to BEM](http://getbem.com/naming/):
> Modifier names may consist of Latin letters, digits, dashes and underscores. CSS class is formed as block’s or element’s name plus two dashes: .block--mod or .block__elem--mod and .block--color-black with .block--color-red. Spaces in complicated modifiers are replaced by dash.

So it should be :
```css
.main-button--get-started {}
```

Reply:
This is really a strange thing...
According to [this source](https://en.bem.info/methodology/naming-convention/):

> The modifier name is separated from the block or element name by a single underscore (_) i.e. block__element_modifier

Where is the truth?

Reply:
Who knows :)
That's strange I [used to](https://css-tricks.com/bem-101/) double dash / double underscore style. BEM was developed by Yandex many years ago. I don't know why but maybe they changed their style. They pushed BEM far from what it was in the beginning. But you don't have to. Class naming convention is more then enough at the beginning and I don't recommend you to fall deep inside it because I haven't seen it was used more then that in production